### PR TITLE
Add notes about anyCastPort and ingressControllerTargetPort

### PR DESCRIPTION
**Pull request by panyuenlau**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?

Corresponding docs for https://github.com/banzaicloud/koperator/pull/997

### Why?

Add docs for the new feature and existing functionality

### CODE REVIEW

**Review by fekete-robert — DISMISSED** [review]

File: docs/external-listener/index.md
Comment (pregnor) [inline docs/external-listener/index.md:42]: **Optional**: We **MAY** consider using the Go style description to make it obvious which field we are talking about.

```suggestion
      # anyCastPort sets which port clients can use to reach all the brokers of the Kafka cluster, default is 29092
      # valid range: 0 < x < 65536
      # this doesn't have impact if using NodePort to expose the Kafka cluster
      anyCastPort: 443
```

File: docs/external-listener/index.md
Comment (pregnor) [inline docs/external-listener/index.md:46]: **Optional**: We **MAY** consider using the Go style description to make it obvious which field we are talking about.

```suggestion
      # ingressControllerTargetPort sets which port the ingress controller uses to handle the external client traffic through the "anyCastPort", default is 29092
      # valid range: 1023 < x < 65536
      # this doesn't have impact if using NodePort to expose the Kafka cluster
      ingressControllerTargetPort: 3000
```

File: docs/external-listener/index.md
Comment (pregnor) [inline docs/external-listener/index.md:76]: **Note**: the comments should be identical to the ones above.

**Review by pregnor — APPROVED** [review]

File: docs/external-listener/index.md
Comment (panyuenlau) [reply docs/external-listener/index.md]: Updated

**Comment by panyuenlau** [comment]
@pregnor @fekete-robert We shouldn't merge this before the next Koperator release is dropped, right?

**Comment by pregnor** [comment]
> @pregnor @fekete-robert We shouldn't merge this before the next Koperator release is dropped, right?

I guess so, but I'm lost what is auto-released and what is not so I defer to Robi's answer.

**Review by pregnor — APPROVED** [review]

**Comment by panyuenlau** [comment]
> > @pregnor @fekete-robert We shouldn't merge this before the next Koperator release is dropped, right?
> 
> I guess so, but I'm lost what is auto-released and what is not so I defer to Robi's answer.

yea... I started to think that we might need to use release tag, so that committing to master branch wouldn't make the commit public

**Comment by fekete-robert** [comment]
@pregnor @panyuenlau : Stuff from the master branch is not published automatically, but new merges automatically open  PR for the banzai site. So we can either mark the PR with something like Do not merge until X.Y release, or we can merge it, and then I'll (hopefully) keep in mind that we shouldn't publish updates to the koperator docs on the banzai site.

This will improve when we migrate the docs and the code to the new github org, where we can have versioned docs, and have a separate set for version X.Y, and also a latest greatest build from the main branch.

**Pull request merged**
